OpenApiSpecs/exchangeSetService_OpenApi_definition_internal.yaml

```diff
@@ -0,0 +1,1291 @@
+              
+                source: https://example.com/storage/tenant/container
+                subject: mynewfile.jpg
+        time:
+          type: string
+          format: date-time
+          description: |
+            Timestamp of when the occurrence happened. If the time of the occurrence cannot be determined then this attribute MAY be set to some other time (such as the current time) by the CloudEvents producer, however all producers for the same source MUST be consistent in this respect. In other words, either they all use the actual time of the occurrence or they all use the same algorithm to determine the value used.
+        data:
+          type: object
+          description: |
+            The event payload. This specification does not place any restriction on the type of this information. It is encoded into a media format which is specified by the datacontenttype attribute (e.g. application/json), and adheres to the dataschema format when those respective attributes are present.
+
+            The `uk.co.admiralty.avcsData.exchangeSetCreated.v1` event payload is described by "#/components/schemas/exchangeSetResponse".
+          allOf:
+            - $ref: "#/components/schemas/exchangeSetResponse"
+      example:
+        specversion: "1.0"
+        type: "uk.co.admiralty.avcsData.exchangeSetCreated.v1"
+        source: " https://exchangeset.admiralty.co.uk/avcsData"
+        id: "2f03a25f-28b3-46ea-b009-5943250a9a41"
+        time: "2021-02-17T14:04:04.4880776Z"
+        subject: "Requested S-100 Exchange Set Created"
+        datacontenttype: "application/json"
+        data:
+          _links:
+            exchangeSetBatchStatusUri:
+              href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272/status"
+            exchangeSetBatchDetailsUri:
+              href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272"
+            exchangeSetFileUri:
+              href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272/files/exchangeset123.zip"
+            errorFileUri:
+              href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272/files/error.txt"
+          exchangeSetUrlExpiryDateTime: "2021-02-17T16:19:32.269Z"
+          requestedProductCount: 4
+          exchangeSetProductCount: 1
+          requestedProductsAlreadyUpToDateCount: 0
+          requestedProductsNotInExchangeSet:
+            [
+              { productName: "101GB40079ABCDEFG", reason: "invalidProduct" },
+            ]
+          fssBatchId: "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272"
+          s100CloudEvent:
```

Comment:
Not completely sure, but this feels like another duplication of the s100CloudEvent here?

Reply:
Duplication hence removed.